bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -70,10 +71,24 @@ verify_images() {
     verify_images
 }
 
+# Remove all the kubectl clients from the .kuberlr directory.
+# Then run `kubectl`, and it should pull in the `kubectl` for
+# the current k8s version in that directory.
+
+verify_kuberlr_for_version() {
+    k8sVersion=$1
+    rm -f "${HOME}"/.kuberlr/"${OS}"-"${ARCH_FOR_KUBERLR}"/kubectl*
+    run kubectl version
+    assert_output --regexp "Client Version.*GitVersion:.v$k8sVersion"
+    run ls -l "$HOME/.kuberlr/${OS}-${ARCH_FOR_KUBERLR}"
+    assert_output --partial "kubectl$k8sVersion"
```

Comment:
Use `bats-file` assert functions to verify a file exists instead of checking `ls -l`  output.

Also consistently use braces; since you wrote `${HOME}` you should also write `${k8s_version}`. Personally I find this even more important for lowercase variable names.

```suggestion
    assert_exists "${kuberlr_cache}/kubectl${k8s_version}"
```